types/google-cloud__storage/README.md

```diff
@@ -0,0 +1,15 @@
+# Installation
```

Comment:
Currently we won't publish a custom README. You could put anything important at the top of `index.d.ts`.